Cleanup setup

cmc333333 opened the pull request:
Analogous to 18F/regulations-core#18

khandelwal commented on setup.py:
```diff
@@ -1,8 +1,13 @@
-    version = "0.1.0", 
-    license = "public domain", 
-    packages = find_packages()
+    name="regulations",
```
Should the name have '-site' in it?

cmc333333 replied:
The app has had the name `regulations` for a long time. I'm inclined to keep it that way for compatibility

khandelwal commented on setup.py:
```diff
@@ -1,8 +1,13 @@
-    packages = find_packages()
+    name="regulations",
+    version="2.0.0", 
+    license="public domain", 
```
You might want to add CC0 as well, as public domain isn't a thing outside the US.

cmc333333 replied:
Sure

cmc333333 commented:
Updated with an explicit CC0 mention. Switched to a "Trove" classifier, per [recommendation](https://docs.python.org/2/distutils/setupscript.html#additional-meta-data)
